src/plugins/orangehrmAdminPlugin/config/routes.yaml

```diff
@@ -558,3 +568,13 @@ admin_register_oauth_client: # remove this
   path: /admin/registerOAuthClient
   controller: OrangeHRM\Core\Controller\Common\ModuleUnderDevelopmentController::handle
   methods: [ GET ]
+
+admin_corporate_branding: # remove this
+  path: /admin/addTheme
+  controller: OrangeHRM\Core\Controller\Common\ModuleUnderDevelopmentController::handle
+  methods: [ GET ]
```

Comment:
delete this route. this seems duplicated, that's why it's failing unit tests

Reply:
Done